SPECS/pmix/pmix.spec

```diff
@@ -111,29 +96,25 @@ find %{buildroot} -name '*.la' | xargs rm -f
 %dir %{_sysconfdir}/%{name}
 %config(noreplace) %{_sysconfdir}/%{name}/*.conf
 %{_datadir}/%{name}/*.txt
-%{_libdir}/libmca_common_dstore.so.1*
 %{_libdir}/libpmix.so.2*
 %{_libdir}/%{name}/*.so
+%{_mandir}/man1/*.1*
 
 %files devel
 %{_datadir}/%{name}/*.supp
 %{_includedir}/pmix*.h
-%{_libdir}/libmca_common_dstore.so
 %{_libdir}/libpmix.so
-
-%files pmi
-%{_libdir}/libpmi.so.1*
-%{_libdir}/libpmi2.so.1*
-
-%files pmi-devel
-%{_includedir}/{pmi,pmi2}.h
-%{_libdir}/libpmi.so
-%{_libdir}/libpmi2.so
+%{_libdir}/pkgconfig/*.pc
 
 %files tools
 %{_bindir}/*
 
 %changelog
+* Thu Feb 02 2023 Riken Maharjan <rmaharjan@microsoft.com> - 4.1.2-1
+- Move from Extended to core
+- Update to 4.1.2
```

Comment:
If we've used Fedora's spec as guidance for the upgrade, let's mention it here as well.

Reply:
updated.